src/Take.Blip.Client/Constants.cs

```diff
@@ -17,5 +17,7 @@ public static class Constants
         public const string POSTMASTER = "postmaster";
 
         public const string DEFAULT_STATE = "default";
+
+        public const string ORIGINAL_SUBFLOW_REDIRECT_FROM = "originalSubflowRedirectFrom";
```

Comment:
Why not `originalRedirectFrom`? No need to couple this functionality to the subflow.

Reply:
not to be confused with other "originals redirect from"